src/main.rs

```diff
@@ -368,6 +368,12 @@ fn app(authors: &'static str, version: &'static str) -> clap::App<'static, 'stat
                         .required(false)
                         .help("Specify the bind address to serve on")
                 )
+                .arg(clap::Arg::with_name("identities")
+                    .long("--identities")
+                    .takes_value(true)
+                    .required(false)
+                    .help("Http request identities, fpm allows these identities to access documents")
```

Comment:
This is a feature only for debugging if FPM is working correctly. Can we do something so this field is not documented, not available as part of `fpm --help`?

Reply:
Also lets ensure this feature is compiled in only if remote feature is not enabled.